spec/app/validators/password_is_strong_validator_spec.rb

```diff
@@ -66,58 +66,43 @@
       password_is_strong_validator.send(:contains_username?, username, password)
     end
 
-    let(:username) do
-      ''
-    end
-
-    context 'with blank password' do
-      let(:password) do
-        ''
+    context 'without blank username' do
+      let(:username) do
+        'username'
       end
 
-      it { is_expected.to eq(false) }
-    end
-
-    context 'without blank password' do
-      let(:password) do
-        'password'
-      end
-
-      context 'with blank username' do
-        let(:username) do
+      context 'with blank password' do
+        let(:password) do
           ''
         end
-
         it { is_expected.to eq(false) }
       end
-
-      context 'without blank username' do
+      context 'with reserved regex characters in username' do
         let(:username) do
-          'username'
+          "user(with)regex"
         end
-
-        it 'should escape username' do
-          expect(Regexp).to receive(:escape).with(username).and_call_original
-
-          contains_username?
+        context 'with reserved regex characters in password' do
```

Comment:
```suggestion
        context 'with username in password' do
```

also please add whitespace lines to separate new contexts and preconditions.